controllers/Endpoint.controller.ts

```
@@ -221,4 +227,171 @@ export class EndpointController {
+        );
+        logApiRequest('/api/deposit/notify', 'POST', req.body.depositKey, logApiData, 400);
+
+        res.status(400).json({
+          success: false,
+          error: 'Invalid request body',
+          details: validationResult.error.flatten(),
+        });
+        return;
+      }
+
+      const { depositKey: depositKeyRaw, fundingTx, reveal, destinationChainDepositOwner, initTxHash, backendAddress } =
```

ESLint

[error] Replace `·depositKey:·depositKeyRaw,·fundingTx,·reveal,·destinationChainDepositOwner,·initTxHash,·backendAddress·}·=` with `⏎········depositKey:·depositKeyRaw,⏎········fundingTx,⏎········reveal,⏎········destinationChainDepositOwner,⏎········initTxHash,⏎········backendAddress,`

(prettier/prettier)